src/main/java/ua/org/crazy/homework02/SortingUtils.java

```diff
@@ -0,0 +1,152 @@
+package ua.org.crazy.homework02;
+
+import java.util.Arrays;
+
+class SortingUtils {
+    //алгоритм сортировки вставками
+    static int[] insertionSort(int[] arr){
+        long start = getCurrentTime();
+        int key;
+        for (int i = 0; i < arr.length; i++) {
+            key = arr[i];
+            int j = i - 1;
+            while (j >= 0 && arr[j] < key){
+                arr[j + 1] = arr[j];
+                j = j - 1;
+            }
+            arr[j + 1] = key;
+        }
+        long finish = getCurrentTime();
+        System.out.println("the work is over for " + (finish - start) + "ms");
+        return arr;
+    }
+
+    //алгоритм сортировки выбором
+    static int[] choiceSort(int[] arr){
+        long start = getCurrentTime();
+        for (int i = 0; i < arr.length; i++) {
+            int minArr = arr[i];
+            int minI = i;
+            for (int j = i + 1; j < arr.length; j++) {
+                if (arr[j] < minArr){
+                    minArr = arr[j];
+                    minI = j;
+                }
+            }
+            if (i != minI){
+                int temp = arr[i];
```

Comment:
It could move to a separate method. For example swap.  Please create it and reuse.